css/style.css

```diff
@@ -0,0 +1,81 @@
+@font-face {
+  font-family: poppins;
+  src: url("https://fonts.googleapis.com/css?family=Poppins");
+}
+
+.header {
+  width: 375px;
+  height: 90px;
+  color: #6070ff;
+  margin: auto;
+  display: flex;
+  justify-content: space-between;
+}
+
+h2 {
+  font-size: 25px;
+}
+
+.material-icons {
+  margin-top: 25px;
+  font-size: 35px;
+}
+
+.container {
+  width: 375px;
+  height: 718px;
+  margin: auto;
+  background-image: url(header-shapes\ mobile@2x.png);
+  background-repeat: no-repeat;
+  border-radius: 0 0 10px 0;
+}
```

Comment:
- [ ] Please make sure that the content of the section is aligned with the `flexbox`. This gives us a lot of options to align all our elements, plus, it's responsive 💪✌
- [ ] Please consider reducing the space between the header and container section. Also, please note that the placement of background image is not according to the given design guidelines. Kindly fix this 💪⛏

Your design | Given design 
---| ---|
![image](https://github.com/jalal1379/Microverse_portflaio/assets/106119105/53c6228d-63af-4917-add0-8d46727887cc) | ![image](https://github.com/jalal1379/Microverse_portflaio/assets/106119105/3591923d-b346-437c-a770-12e39af97fb1)